### ISSUE TITLE
Slovenský název filmu místo českého

### ISSUE THREAD

**Issue by drSchaps** [ekarorgit/metadata.csfd.cz#19]
U některých filmů mám problém s tím, že se zobrazí slovenský název filmu místo českého. Po kratším hledání jsem zjistil, že se to děje u filmů, které mají u názvu filmu českou vlaječku a jako tooltip "Československo".
Např. 
https://www.csfd.cz/film/2665-vetrelec-3/prehled/
https://www.csfd.cz/film/32110-cetnik-a-cetnice/prehled/
https://www.csfd.cz/film/166-jmeno-ruze/prehled/

![image](https://user-images.githubusercontent.com/59886963/72366081-d6fec980-36f9-11ea-9c82-b7133b31ec68.png)
Mohu poprosit o opravu?

Jinak děkuji všem vývojářům, kteří se podílejí na tomto pluginu!

**Comment by ekarorgit** [comment]
@drSchaps 
No, jde o to ze plugin ma uplne stejne informace co jsou na webu, takze poznat kterej nazev je ceskej je dost obtizne a delam to v zasade jen podle vlajky. A kdyz daj k ceske (ceskoslovenske) vlajce, cesky nebo slovensky nazev, tak to nemam jak rozeznat.
Jedina rada bude v pluginu misto volby jazyku "cs" zkusit dat "local". To bere uplne prvni nazev na strance co tam je. Volba "original" dava druhej a "cs" dava prvni s ceskou vlajkou (pokud neni tak da prvni)
Co jsem vypozoroval tak ten uplne prvni by mel castecne brat v potaz odkud se pripojujete, takze v Cechach by Vam mel dat ceskej nazev, na Slovensku slovenskej.

Volby jazyka jsou trochu popsany v changelogu. 
https://github.com/ekarorgit/metadata.csfd.cz/blob/master/changelog.txt#L15

**Comment by tibmattm** [comment]
Dobry den chcem sa opytat, CSFD stranka presla zmenou a teraz nieje mozne nacitat do kniznice ziadne filmy bude sa tento problem dat nejako vyriesit? Bol by som velmi rad a urcite nebudem sam ak by to bolo mozne, vopred dakujem.
